packages/react-components/src/components/ChatMessage/ChatMessageComponentAsEditBox.tsx

```diff
@@ -66,7 +73,7 @@ export const ChatMessageComponentAsEditBox = (props: ChatMessageComponentAsEditB
   }, []);
 
   const setText = (
-    event: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>,
+    event: React.FormEvent<HTMLInputElement | HTMLTextAreaElement> | null,
```

Comment:
Are we allowed to assign `null` now? Can you explain a little bit more why this case is needed?

Reply:
Null is needed as sometimes there is no event that can be sent on this API.

Reply:
Can we make it undefined?
Or even
event?: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>